pages/express-relay/contract-addresses/svm.mdx

```diff
@@ -3,26 +3,75 @@ import AddressSvmTable from "../../../components/AddressSvmTable";
 
 Express Relay is currently deployed on the following SVM environments:
 
-<Tabs items={['Solana Mainnet-Beta Testing']}>
+<Tabs items={['Prod (Solana Mainnet-Beta)', 'Testing (Solana Mainnet-Beta)']}>
+<Tabs.Tab>
+Auction Server endpoint: https://pyth-express-relay-mainnet.asymmetric.re/
+
+### Prod
+
+**Network Details:**
+
+<AddressSvmTable
```

Comment:
Can you add `copy to clipboard` functionality as well. You can use `/components/CopyAddress.tsx` as reference.